css-canvas-medical/styles.css

```diff
@@ -0,0 +1,209 @@
+html {
+  font-family: 'tiempos headline', sans-serif;
+  scroll-behavior: smooth;
+}
+
+p,
+a,
+button {
+  font-family: 'Avenir', sans-serif;
+}
+
+h2 {
+  color: #929292;
+}
+
+header {
+  display: flex;
+  align-items: center;
+  padding-top: 16px;
+  padding-bottom: 16px;
+  padding-left: 80px;
+  padding-right: 80px;
+  border-bottom: 1px solid #f2f4f5;
+}
+
+.blue-button {
+  border: none;
+  background: #2a4464;
+  color: white;
+  padding: 12px 30px;
+  padding-bottom: 12px;
+  padding-left: 30px;
+  padding-right: 30px;
+  border-radius: 4px;
+}
+
+header img {
+  height: 26px;
+}
+
+header nav a {
+  margin-left: 50px;
+  text-decoration: none;
+}
+
+header nav .blue-button {
+  padding-top: 10px;
+  padding-bottom: 10px;
+  padding-left: 16px;
+  padding-right: 16px;
+  margin-left: 18px;
+}
+
+.justify-between {
+  justify-content: space-between;
+}
+
+.nav-item {
+  color: #a7a7a7;
+  font-size: 15px;
+}
+
+.container {
+  display: flex;
+  flex-direction: column;
+}
+
+.row.hero-image {
+  margin: 60px;
+  position: relative;
+  display: flex;
+  flex-direction: column;
+  overflow: hidden;
+}
```

Comment:
Instead of making this dependent on the `row` class, you should just make this a separate class you can add to the element